From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-memory_hotplug-export-mhp_supports_memmap_on_memory-fix
Date: Sat Jan 27 07:13:46 PM PST 2024

fix build

In file included from ./include/linux/mmzone.h:1425,
                 from ./include/linux/gfp.h:7,
                 from ./include/linux/slab.h:16,
                 from ./include/linux/crypto.h:17,
                 from arch/x86/kernel/asm-offsets.c:9:
./include/linux/memory_hotplug.h:282:13: warning: 'mhp_supports_memmap_on_memory' defined but not used [-Wunused-function]
  282 | static bool mhp_supports_memmap_on_memory(void)
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Cc: Vishal Verma <vishal.l.verma@intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/memory_hotplug.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/memory_hotplug.h~mm-memory_hotplug-export-mhp_supports_memmap_on_memory-fix
+++ a/include/linux/memory_hotplug.h
@@ -279,7 +279,7 @@ static inline bool movable_node_is_enabl
 	return false;
 }
 
-static bool mhp_supports_memmap_on_memory(void)
+static inline bool mhp_supports_memmap_on_memory(void)
 {
 	return false;
 }
_
